nextjs-app/components/pages/program-rules/Schedule/ScheduleTabs.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+
+import { useState } from "react";
+import Link from "next/link";
+import Button from "@/components/common/Button";
+import TabToggle from "./TabToggle";
+import TabContentSailor from "./TabContentSailor";
+import TabContentNavigator from "./TabContentNavigator";
+
+const SCHEDULE_TAB_DATA = [
+  {
+    key: "sailor",
+    label: "水手",
+    child: <TabContentSailor key="sailor" />,
```

Comment:
```suggestion
    content: <TabContentSailor  />,
```
prefer calling it `content`, and why do we need key prop here ?